Formula/l/lima.rb

```diff
@@ -22,12 +22,25 @@ class Lima < Formula
     depends_on "qemu"
   end
 
+  # PR https://github.com/lima-vm/lima/pull/3564
+  # For https://github.com/Homebrew/homebrew-core/pull/224206#issuecomment-2896811998
+  # > Minitest::Assertion: Expected
+  # > {"aarch64" => {"location" => "/home/linuxbrew/.linuxbrew/Cellar/lima/1.1.0/share/lima/
+  # > lima-guestagent.Linux-aarch64.gz"}} to include "riscv64".}
+  patch do
+    url "https://github.com/lima-vm/lima/commit/230e36c359aa999c24f8a8c15045877342b2f231.patch?full_index=1"
+    sha256 "e4728c62d228b5f3c678ff85c723af0918b65dc8d1d777f1ea29fe619b0ff9b3"
+  end
+
   def install
+    # make (default):              build everything
+    # make native:                 build core + native guest agent
+    # make additional-guestagents: build non-native guest agents
```

Comment:
This will break when we need to more components that are not guestagent. Would be more future proof if we call this "extras".

Reply:
The current `native` package is not splittable to smaller ones.
When the situation changes, new Makefile targets will be added with the best effort to minimize the effects on compatibility.